partner_communication_switzerland/models/contracts.py

```diff
@@ -266,8 +266,31 @@ def _send_reminders_for_birthday_in_1day_or_2months(self):
         tomorrow = today + relativedelta(days=1)
         sponsorships_with_birthday_tomorrow = \
             self._get_sponsorships_with_child_birthday_on(tomorrow)
+
+        sponsorships_to_avoid = {}
```

Comment:
Empty recordset should be initialised like this:
```suggestion
        sponsorships_to_avoid = self.env['recurring.contract']
```